lite/api/contents/all.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Should this file be in the repo? Same question for `.jupyterlite.doit.db`. They seem like temp files?

Reply:
@ino 

1. The `all.json` file in the content folders actually generates automatically during the build process when we run:
`jupyter lite build`.
Actually behind the scenes:
  
    _The build process indexes the contents found in {lite-dir}/files/ or any content roots specified via the CLI flag --contents or in jupyter_lite_config.json under #/LiteBuildConfig/contents. The indexed content is then saved in the output directory under {output-dir}/api/contents/{subdir?}/all.json._
    
    Refrence: https://jupyterlite.readthedocs.io/en/latest/howto/content/files.html#content-with-the-cli

2. **But we can definately delete the** `.jupyterlite.doit.db`. It is a temp file.

Reply:
@inodb I suppose we cannot delete the .jupyterlite.doit.db also.

ref: https://discourse.jupyter.org/t/jupyter-lite-json-not-taken-into-account/21717.

I experinced the same issue.

There is also an open issue in jupyterlite regarding the same: https://github.com/jupyterlite/jupyterlite/issues/1194

Reply:
ok thanks for checking and explaining @gautamsarawagi !